src/sentry/receivers/outbox/control.py

```diff
@@ -127,3 +130,47 @@ def process_issue_email_reply(shard_identifier: int, payload: Any, **kwds):
         from_email=payload["from_email"],
         text=payload["text"],
     )
+
+
+# See the comment on /src/sentry/tasks/relocation.py::uploading_start for a detailed description of
+# how this outbox drain handler fits into the entire SAAS->SAAS relocation workflow.
+@receiver(process_control_outbox, sender=OutboxCategory.RELOCATION_EXPORT_REQUEST)
+def process_relocation_request_new_export(payload: Mapping[str, Any], **kwds):
+    encrypt_with_public_key = (
+        payload["encrypt_with_public_key"].encode("utf-8")
```

Comment:
What would be the reason we expect other types of payloads besides strings?

Reply:
The hybrid cloud (or pydantic?) machinery seems to serialize them to strings when sending them over the wire. Not sure why this happens, but this seems to resolve the issue.